runtime/common/src/main/java/edu/snu/nemo/runtime/common/RuntimeIdGenerator.java

```diff
@@ -76,36 +79,25 @@ public static String generateStageId(final Integer stageId) {
   }
 
   /**
-   * Generates the ID for {@link edu.snu.nemo.runtime.common.plan.physical.Task}.
-   *
-   * @param irVertexId the ID of the IR vertex.
-   * @return the generated ID
-   */
-  public static String generateLogicalTaskId(final String irVertexId) {
-    return "Task-" + irVertexId;
-  }
-
-  /**
-   * Generates the ID for {@link edu.snu.nemo.runtime.common.plan.physical.Task}.
+   * Generates the ID for the physical representation of an IRVertex.
    *
-   * @param index         the index of the physical task.
-   * @param logicalTaskId the logical ID of the task.
+   * @param irVertexId the id of the IRVertex.
+   * @param index      the index of the physical task.
    * @return the generated ID
    */
-  public static String generatePhysicalTaskId(final int index,
-                                              final String logicalTaskId) {
-    return logicalTaskId + PHYSICAL_TASK_ID_SPLITTER + index;
+  public static String generatePhysicalIRVertexId(final String irVertexId,
+                                                  final int index) {
+    return irVertexId + PHYSICAL_IRVERTEX_ID_SPLITTER + index;
   }
 
   /**
-   * Generates the ID for {@link edu.snu.nemo.runtime.common.plan.physical.ScheduledTask}.
+   * Generates the ID for a task.
    *
    * @param index   the index of this task.
    * @param stageId the ID of the stage.
    * @return the generated ID
    */
-  public static String generateTaskId(final int index,
-                                           final String stageId) {
+  public static String generateTaskId(final int index, final String stageId) {
```

Comment:
This method makes it look like the tasks still exist. Please add a comment on the PR to better illustrate what this PR does, as it doesn't seem like it's getting rid of the tasks after all.

Reply:
This one is for the 'new' Task (formerly TaskGroup). I've updated the PR message to make it clear that the changes only remove the 'old' Task.